features/fout-meldingen.feature

```diff
@@ -0,0 +1,168 @@
+#language: nl
+Functionaliteit: Gebruiksvriendelijke foutmeldingen
+
+    Als developer van een consumer applicatie
+    Wil ik gebruiksvriendelijke en duidelijke foutmeldingen
+    Zodat ik deze één op één kan gebruiken in de UI van mijn applicatie
+
+Rule: minimaal één optionele parameter is opgegeven
+
+    Scenario: Er zijn geen parameters opgegeven
+        Als '/wozobjecten' wordt aangeroepen
+        Dan bevat de response de volgende kenmerken
+        | naam   | waarde                                                                                                                                                                         |
+        | title  | Tenminste één parameter moet worden opgegeven.                                                                                                                                 |
+        | status | 400                                                                                                                                                                            |
+        | detail | Er moet minimaal één van de parameters 'rsin', 'kvkNummer', 'adresseerbaarObjectIdentificatie', 'nummeraanduidingIdentificatie' of 'postcode' en 'huisnummer' worden opgegeven |
```

Comment:
Omdat postcode altijd in combinatie met huisnummer moet worden opgegeven, zou ik zeggen:
Je moet minimaal één van de parameters 'rsin', 'kvkNummer', 'adresseerbaarObjectIdentificatie', 'nummeraanduidingIdentificatie' opgeven. 'Postcode' in combinatie met 'huisnummer' kan ook.